geomstats/geometry/connection.py

```diff
@@ -627,6 +627,64 @@ def path(t):
 
         return path
 
+    def _geodesic_ivp(self, initial_point, initial_tangent_vec):
+        """Solve geodesic initial value problem.
+
+        Compute the parameterized function for the geodesic starting at
+        initial_point with initial velocity given by initial_tangent_vec.
+
+        Parameters
+        ----------
+        initial_point : array-like, shape=[..., dim]
+            Initial point.
+
+        initial_tangent_vec : array-like, shape=[..., dim]
+            Tangent vector at initial point.
+
+        Returns
+        -------
+        path : function
+            Parameterized function for the geodesic curve starting at
+            initial_point with velocity initial_tangent_vec.
+        """
+        if _check_exp_solver(self, raise_=False) and hasattr(
```

Comment:
Slightly urelated: could you add docstrings into the numerics and the solvers? I know what most functions are private thus docstrings are not strictly needed. Yet, I find myself having problems to understand (and use) the code.